const_test.go

```diff
@@ -0,0 +1,31 @@
+package gltf
+
+import (
+	"encoding/json"
+	"testing"
+)
+
+func TestAccessorType_UnmarshalJSON(t *testing.T) {
+	tests := []struct {
+		name    string
+		expType AccessorType
+		typeStr []byte
+		wantErr bool
+	}{
+		{"base", AccessorVec3, []byte(`"VEC3"`), false},
+		{"incorrect-type", 100, []byte(`"CUSTOM_TYPE"`), true},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			var accType AccessorType = 100
```

Comment:
this should be a test parameter

Reply:
Thank you, added test arguments - I hope I have understood you correctly